src/main/java/hello/commerce/order/OrderService.java

```diff
@@ -0,0 +1,26 @@
+package hello.commerce.order;
+
+import hello.commerce.order.dto.OrderResponseDtoV1;
+import hello.commerce.order.dto.OrderListResponseDtoV1;
+import hello.commerce.order.model.OrderStatus;
+
+import java.awt.print.Pageable;
+
+public interface OrderService {
+
+    /**
+     * 주문 목록 조회
+     * @param pageable
+     * @param orderStatus
+     * @return 주문 목록 응답
+     */
+    OrderListResponseDtoV1 getOrders(Pageable pageable, OrderStatus orderStatus);
```

Comment:
model/Order.class

하나 정의해서 아래처럼 인터페이스 관리하는게 더 좋은 방향입니다.(객체지향적으로 더불어 레이어별 디펜던시를 줄임으로써 유지보수성이 좋아집니다)

```suggestion
    List<Order> getOrders(Pageable pageable, OrderStatus orderStatus);
```

Reply:
dto 사용 및 변환은 컨트롤러 레이어를 넘어서지 않도록 하겠습니다. 해당 부분 좀 더 공부해보겠습니다.